models/evc.py

```diff
@@ -292,7 +313,8 @@ def as_dict(self):
         evc_dict["sb_priority"] = self.sb_priority
         evc_dict["service_level"] = self.service_level
         evc_dict["primary_constraints"] = self.primary_constraints
-        evc_dict["secondary_constraints"] = self.secondary_constraints
+        evc_dict["flow_removed_at"] = self.flow_removed_at
```

Comment:
@gretelliz I believe you removed this accidentally (I end up creating an issue for that https://github.com/kytos-ng/mef_eline/issues/236)

```suggestion
        evc_dict["secondary_constraints"] = self.secondary_constraints
        evc_dict["flow_removed_at"] = self.flow_removed_at
```